<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>137</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Save to:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="mFileName"/>
      </item>
      <item>
       <widget class="QLabel" name="mAckNum">
        <property name="toolTip">
         <string>format: left[ack msg]</string>
        </property>
        <property name="text">
         <string>00000</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="mReack">
        <property name="text">
         <string>ReAck</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="mFileBrowserBtn">
        <property name="text">
         <string>Browser</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="Tips">
      <property name="text">
       <string>If transfer been blocked, please press ReAck button</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="mProcessBar">
      <property name="value">
       <number>0</number>
      </property>
      <property name="format">
       <string notr="true">%p%</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>521</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
